WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/storage/PersisterBase.java

```diff
@@ -18,17 +18,17 @@
  * WatchDog instance can record before the database breaks is
  * {@link Long#MAX_VALUE}.
  */
-public abstract class PersisterBase<T extends WatchDogTransferable> {
+public class PersisterBase {
 
 	private boolean isClosed;
 
 	/** The name of the DB collection to be used. */
-	private final String collection;
+	private static final String COLLECTION = "watchdog";
```

Comment:
In the specific wrappers for Events/Intervals, so in IntervalPersisterBase and EventPersisterBase.